### PR TITLE
Upload to blob container fix

### PR DESCRIPTION
**Pull request by sahuroshan**
updated the node module version to fix the issue. This seems to be known issue with the older versions of azure/storage-blob module.

found similar issue here - https://github.com/Azure/azure-sdk-for-js/issues/28059#issuecomment-1971053922

ADO bug link - https://msdata.visualstudio.com/Database%20Systems/_workitems/edit/3044065

### CODE REVIEW

File: extensions/sql-migration/package.json [hunk 1]
```diff
@@ -178,7 +178,8 @@
     "@microsoft/ads-service-downloader": "^1.2.1",
     "@microsoft/ads-extension-telemetry": "^3.0.1",
     "uuid": "^8.3.2",
-    "vscode-nls": "^4.1.2"
+    "vscode-nls": "^4.1.2",
+    "@azure/storage-blob": "^12.23.0-beta.1"
```
Comment (Ramudaykumar) [inline extensions/sql-migration/package.json:182]: Have tested for any regression, as we are using Storage account for read the blob many other places well. Just test for a complete File share migration, and a blob migration.

**Review by Ramudaykumar — APPROVED** [review]
Approving with a comment to test other blob scenarios.

**Comment by Ramudaykumar** [comment]
Build has also failed, check if it is due to our change.

File: extensions/sql-migration/package.json (hunk 1 above)
Comment (Charles-Gagnon) [inline extensions/sql-migration/package.json:182]: You need to run `yarn` (or `yarn install`) and then commit the updated lockfille as well

Comment (sahuroshan) [reply extensions/sql-migration/package.json:182]: at other places the utility functions related to storage account are referred from azure core extension, the module that I added is for my change only.

Comment (sahuroshan) [reply extensions/sql-migration/package.json:182]: done

**Pull request merged**